WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogListItemViewModel.kt

```diff
@@ -14,20 +14,31 @@ data class ActivityLogListItemViewModel(
     private val status: String?,
     val isRewindable: Boolean,
     val rewindId: String?,
-    private val datePublished: Date
+    private val datePublished: Date,
+    val isRewindInProgress: Boolean = false
 ) {
-    private val timeFormatter = DateFormat.getDateInstance(DateFormat.LONG, Locale.getDefault())
-
     companion object {
         @JvmStatic
         fun fromDomainModel(model: ActivityLogModel): ActivityLogListItemViewModel {
             return ActivityLogListItemViewModel(model.activityID, model.summary, model.text, model.gridicon,
                     model.status, model.rewindable ?: false, model.rewindID, model.published)
         }
+
+        fun makeRewindItem(title: String, message: String): ActivityLogListItemViewModel {
```

Comment:
I would consider having two different classes for the UiModel with the same sealed ancestor. For example `sealed class ActivityLogListItem` would have 2 children, `RewindListItem` and `ActivityListItem`. They can be handled differently by checking `is` and they could have different layouts. It would make it simple to add more types in future (we're expecting other announcements in the list).